lighthouse-core/gather/gatherers/http-redirect.js

```diff
@@ -27,48 +28,12 @@ class HTTPRedirect extends Gatherer {
 
   constructor() {
     super();
-    this._preRedirectURL = undefined;
-  }
-
-  beforePass(options) {
-    this._preRedirectURL = options.url;
-    options.url = this._preRedirectURL.replace(/^https/, 'http');
```

Comment:
we still need this part though, that's what lets us test the redirect when an https url is given :)